<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgFacturas</class>
 <widget class="QWidget" name="wdgFacturas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lbl">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Facturas</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lblBusqueda">
         <property name="text">
          <string>Buscar Artículo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtBusqueda"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="table">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Id Facturas</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Hora</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Cliente</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Artículos</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Total venta</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Pagada</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblTotal">
       <property name="text">
        <string>Total artículos: 0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionFacturaClienteGenerico">
   <property name="icon">
    <iconset resource="../images/gnuoptics.qrc">
     <normaloff>:/gnuoptics/factura.png</normaloff>:/gnuoptics/factura.png</iconset>
   </property>
   <property name="text">
    <string>Factura a cliente genérico</string>
   </property>
  </action>
  <action name="actionFacturaModificar">
   <property name="icon">
    <iconset resource="../images/gnuoptics.qrc">
     <normaloff>:/gnuoptics/document-edit.png</normaloff>:/gnuoptics/document-edit.png</iconset>
   </property>
   <property name="text">
    <string>Modificar factura</string>
   </property>
  </action>
  <action name="actionFacturaBorrar">
   <property name="icon">
    <iconset resource="../images/gnuoptics.qrc">
     <normaloff>:/gnuoptics/list-remove.png</normaloff>:/gnuoptics/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>Borrar factura</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../images/gnuoptics.qrc"/>
 </resources>
 <connections/>
</ui>
